Cargo.toml

```diff
@@ -24,17 +24,18 @@ libc = "0.2.73"
 log = "0.4.6"
 percent-encoding = "2.1.0"
 rspotify = "0.8.0"
-serde = { version = "1.0.115", features = ["derive"] }
-serde_ini = "0.2.0"
-sha-1 = "0.9"
-structopt = "0.3.17"
+serde = { version = "1.0.103", features = ["derive"] }
+sha-1 = "0.8"
+structopt = "0.3.11"
 syslog = "4"
 tokio-core = "0.1"
 tokio-io = "0.1"
 tokio-signal = "0.1"
 url = "1.7"
 xdg = "2.2"
 librespot = { version = "0.1.1", default-features = false, features = ["with-tremor"] }
+toml = "0.5.6"
+serde_repr = "0.1.5"
```

Comment:
In this case, I'd prefer a manual `Deserialize` implementation. It's easier for a dev to understand

```rust
impl<'de> Deserialize<'de> for Bitrate {
    fn deserialize<D: de::Deserializer<'de>>(deserializer: D) -> Result<Self, D::Error> {
        match u16::deserialize(deserializer) {
            96 => Ok(Self::Bitrate96),
            160 => Ok(Self::Bitrate160),
            320 => Ok(Self::Bitrate320),
            unknown => Err(D::Error::invalid_value(
                de::Unexepected::Unsigned(unknown.try_into().unwrap()),
                "a bitrate"
            )),
        }
    }
}

Reply:
done